RecoEgamma/EgammaTools/python/calibratedEgammas_cff.py

```diff
@@ -63,3 +67,11 @@
 
 def prefixName(prefix,name):
     return prefix+name[0].upper()+name[1:]
+
+
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+run2_miniAOD_80XLegacy.toModify(calibratedElectrons,correctionFile = _correctionFile2016Legacy)
```

Comment:
it is more straightforward to modify calibratedEgammaSettings, which requires only one line and will simplify maintenance if some other code includes and clones from PSets derived from calibratedEgammaSettings
```run2_miniAOD_80XLegacy.toModify(calibratedEgammaSettings, correctionFile = _correctionFile2016Legacy)```
this line should be placed right after calibratedEgammaSettings is defined.